pkg/sensors/components/source/rapl_sysfs.go

```diff
@@ -51,7 +50,6 @@ var (
 
 func init() {
 	eventPaths = map[string]map[string]string{}
-	detectEventPaths()
```

Comment:
removing `detectEventPaths` from init will leave `eventPaths` uninitialized. can we call `detectEventPaths` in `once.Do` inside the functions which reads `detectEventPaths`

Reply:
same comment for other places.